zubhub_frontend/zubhub/src/views/create_activity/createActivityScripts.js

```diff
@@ -0,0 +1,474 @@
+import * as Yup from 'yup';
+import ZubhubAPI from '../../api';
+import { toast } from 'react-toastify';
+import { nanoid } from 'nanoid';
+import {
+  s3 as DO,
+  doConfig,
+  Compress,
+  slugify,
+} from '../../assets/js/utils/scripts';
+
+const API = new ZubhubAPI();
+
+const vars = {
+  activityImages_field_touched: false,
+  inspiringExemplesImages_field_touched: false,
+  upload_in_progress: false,
+};
+
+// ^/file validation functions
+const isImage = (value) => {
+  if (value) {
+    let not_an_image = false;
+    for (let index = 0; index < value.length; index++) {
+      if (value[index].type.split('/')[0] !== 'image') {
+        not_an_image = true;
```

Comment:
Do we need to check for all the images or can we fail if one of the validation fails